docs/docs/config/behaviors.md

```diff
@@ -182,6 +191,29 @@ You can use the following nodes to tweak the default behaviors:
 | -------- | ----------------------------------------- |
 | `&gresc` | [Grave escape](../behaviors/mod-morph.md) |
 
+## Sensor Rotation
+
+Creates a custom behavior which sends a tap of other behaviors when a sensor is rotated.
+Has two variants: with `compatible = "zmk,behavior-sensor-rotate"` it accepts no parameters when used, whereas with `compatible = "zmk,behavior-sensor-rotate-var"` it accepts two parameters.
+
+See the [sensor rotation behavior](../behaviors/sensor-rotate.md) documentation for more details and examples.
+
+### Devicetree
+
+Definition files:
+[zmk/app/dts/bindings/behaviors/zmk,behavior-sensor-rotate.yaml](https://github.com/zmkfirmware/zmk/blob/main/app/dts/bindings/behaviors/zmk%2Cbehavior-sensor-rotate.yaml),
+[zmk/app/dts/bindings/behaviors/zmk,behavior-sensor-rotate-var.yaml](https://github.com/zmkfirmware/zmk/blob/main/app/dts/bindings/behaviors/zmk%2Cbehavior-sensor-rotate-var.yaml)
+
+| Property                | Type          | Description                                                                                                                                                                        | Default |
+| ----------------------- | ------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- |
+| `label`                 | string        | Unique label for the node                                                                                                                                                          |         |
+| `compatible`            | string        | Sensor rotation type, **must be _one_ of**:<br/>• `"zmk,behavior-sensor-rotate"`<br/>• `"zmk,behavior-sensor-rotate-var"`                                                          |         |
```

Comment:
You can use `<ul>` and `<li>` tags instead of `<br>` and bullet characters to get an actual list here.

Reply:
Also modified this, I had copied from macros without thinking about it much.